eo-maven-plugin/src/main/java/org/eolang/maven/hash/CommitHashesText.java

```diff
@@ -23,65 +23,24 @@
  */
 package org.eolang.maven.hash;
 
-import com.jcabi.log.Logger;
-import java.io.IOException;
-import java.net.URL;
 import org.cactoos.Text;
 import org.cactoos.text.Sticky;
-import org.cactoos.text.TextOf;
-import org.cactoos.text.UncheckedText;
 
 /**
  * Commit hashes table as text from objectionary.
+ * This class serves the purpose of the global cache in order to avoid
+ * downloading the list of tags multiple times from objectionary.
  *
  * @since 0.29.6
- * @todo #1602:30min Come up with a good name for the class. There are many
- *  things we want to say with the name of the class: 1) it's commit hashes
- *  2) it's a text 3) it's loaded from the objectionary. The result name will be
- *  ObjectionaryCommitHashesText and it may look a bit verbose. Maybe it really
- *  does not but we should decide anyway.
  */
 public final class CommitHashesText implements Text {
     /**
      * Cache.
      */
-    private static final Text CACHE = new Sticky(CommitHashesText.load());
-
-    /**
-     * Tags.
-     */
-    private static final String HOME = "https://home.objectionary.com/tags.txt";
+    private static final Text CACHE = new Sticky(new ObjectionaryCommitHashes().load());
```

Comment:
@volodya-lombrozo If `ObjectionaryCommitHashes` will extend `TextEnvelope` (see below), then it will be 
```java
private static final Text CACHE = new Sticky(new ObjectionaryCommitHashes())
```

Reply:
@maxonfjvipon Good suggestion, thanks